packages/itwinui-react/src/core/Buttons/Button/Button.tsx

```diff
@@ -63,15 +78,32 @@ export const Button = React.forwardRef((props, ref) => {
       {...rest}
     >
       {startIcon && (
-        <Box as='span' className='iui-button-icon' aria-hidden>
+        <Box
+          as='span'
+          aria-hidden
+          {...startIconProps}
+          className={cx('iui-button-icon', startIconProps?.className)}
+        >
           {startIcon}
         </Box>
       )}
 
-      {children && <span>{children}</span>}
+      {children && (
+        <span
+          {...labelProps}
+          className={cx('iui-button-label', labelProps?.className)}
```

Comment:
why was this random `iui-button-label` class added?

Reply:
I thought that it would need a default className in case the user didn't specify one using labelProps. But if a default class isn't needed, then I can just remove it.

Reply:
removed the class